Revit_Core_Engine/Query/DrawingArea.cs

```diff
@@ -0,0 +1,110 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using Autodesk.Revit.DB;
+using BH.Engine.Geometry;
+using BH.oM.Adapters.Revit.Settings;
+using BH.oM.Base.Attributes;
+using BH.oM.Geometry;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Compute the outline of Title Block drawing area.")]
+        [Input("document", "Revit current document to be processed.")]
+        [Input("titleBlock", "Title Block symbol to get the outline drawing area from.")]
+        [Output("outline", "Title Block drawing area.")]
+        public static Outline DrawingArea(this Document document, FamilySymbol titleBlock)
+        {
+            Document familyDoc = document.EditFamily(titleBlock.Family);
+            List<DetailLine> lines = new FilteredElementCollector(familyDoc).OfCategory(Autodesk.Revit.DB.BuiltInCategory.OST_Lines).WhereElementIsNotElementType().Select(x => x as DetailLine).ToList();
```

Comment:
```suggestion
            List<DetailLine> lines = new FilteredElementCollector(familyDoc).OfCategory(BuiltInCategory.OST_Lines).WhereElementIsNotElementType().Cast<DetailLine>().ToList();
```

Reply:
Couldn't remove `Autodesk.Revit.DB` because there is already `BuiltInCategory` method in Revit_Tools